src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt

```diff
@@ -56,7 +55,6 @@ class CodyToolWindowContent(private val project: Project) {
     tabbedPane.insertSimpleTab("Chat", chatContainerPanel, CHAT_TAB_INDEX)
     tabbedPane.insertSimpleTab("Chat History", historyTree, HISTORY_TAB_INDEX)
     tabbedPane.insertSimpleTab("Commands", commandsPanel, COMMANDS_TAB_INDEX)
-    tabbedPane.insertSimpleTab("My Account", myAccountPanel, MY_ACCOUNT_TAB_INDEX)
```

Comment:
Let's hide it by default so an enterprise user will not be confused by the appearing and disappearing tab at the IDE start up.